Formula/tmux.rb

```diff
@@ -50,12 +41,7 @@ def install
 
     system "make", "install"
 
-    if build.head?
-      pkgshare.install "example_tmux.conf"
-    else
-      bash_completion.install "examples/bash_completion_tmux.sh" => "tmux"
-      pkgshare.install "examples"
-    end
```

Comment:
The `example_tmux.conf` file [still exists](https://github.com/tmux/tmux/blob/9a4b45dc0fdefead3fe9d471e144da78163860d0/example_tmux.conf), so the `pkgshare.install "example_tmux.conf"`line can be kept.